common/utils/file.go

```diff
@@ -1,48 +0,0 @@
-package utils
-
-import (
-	"crypto/sha256"
-	"io"
-	"os"
-
-	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-)
-
```

Comment:
Would expect create / write / update / delete features in `file`. Might be added later

Reply:
Partially addressed by https://github.com/DataDog/test-infra-definitions/pull/9/files#diff-e5546b99618f257717e785d711dfc4ae15a8d29cc820f60659a75e88d868769b, can be edited on needs